client/src/components/footer/Footer.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import { NavLink } from "react-router-dom";
+import classes from "./footer.module.css";
+import useHttp from "../../hooks/use-http";
+const Footer = () => {
+  const { requestHandler } = useHttp();
+  const [footerAsset, setFooterAsset] = useState("");
+  const contentAssetHandler = (data) => {
+    const modifiedContent = data.c_body.replace(
+      /<ul class="socialIcons-asset">/g,
+      '<ul class="socialIcons-asset ' + classes.socialIcons + '">'
+    );
+    setFooterAsset(modifiedContent);
+  };
+  useEffect(() => {
+    requestHandler(
+      {
+        url: "https://zydc-006.dx.commercecloud.salesforce.com/s/Sites-RefArch-Site/dw/shop/v21_10/content/react-footer?client_id=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
```

Comment:
Please use a config file to keep reference of the SB URL, OCAPI endpoints and client ID.